Easydict/Feature/Configuration/NewConfiguration.swift

```diff
@@ -0,0 +1,595 @@
+//
+//  NewConfiguration.swift
+//  Easydict
+//
+//  Created by ljk on 2024/1/2.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import Defaults
+import Foundation
+
+class Configuration: NSObject {
+    private(set) static var shared = Configuration()
+
+    var appDelegate = NSApp.delegate as? AppDelegate
+
+    var updater: SPUUpdater? {
+        appDelegate?.updaterController.updater
+    }
+
+    @Default(.firstLanguage)
```

Comment:
`@Defaults.Default` property wrapper should be only used in SwiftUI since it introduce a `SwiftUI.ObservedObject` inside for UI tracking which we don't need here.